### PR TITLE
added missing parameter to documentation

### PR DESCRIPTION
**Pull request by enrike1983**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,11 +62,12 @@ To create a `MailingList` you can follow this example. Please, refer to the [Mai
 use Fazland\MailUpRestClient\MailingList;
 
 $email = "owner_of_the_list@email.com";
+$name = "owner_name";
```
Comment (massimilianobraglia) [inline README.md]: It's not the owner name but the list name :)
Just a little clarification for the docs.

**Review by massimilianobraglia — CHANGES_REQUESTED** [review]
Hey there @enrike1983! 
Thank you for your contribution. I just left a little comment to explain better the parameters of that function.

File: README.md (hunk 1 above)
Comment (enrike1983) [reply README.md]: Ok looks good, thank you

**Comment by massimilianobraglia** [comment]
@enrike1983 why did you close this one?

File: README.md (hunk 1 above)
Comment (massimilianobraglia) [reply README.md]: Can you adjust the code?

**Comment by massimilianobraglia** [comment]
Thank you, @enrike1983.

**Pull request merged**